code/modules/antagonists/_common/antag_team.dm

```diff
@@ -60,3 +69,36 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 
 
 	return "<div class='panel redborder'>[report.Join("<br>")]</div>"
+
+/datum/team/proc/get_team_antags(antag_datum, include_subtypes = TRUE)
+	var/datum/list = list()
```

Comment:
im confused why don't you just do 
```
var/list/antag_list = list()
```
or something why `var/datum/list`?

Reply:
because i was sleep deprived, fixed it now.